### DISCUSSION TITLE
Outlook sync failed - JiraAssistant from Jira Applications

### DISCUSSION THREAD

**Discussion by tonisc** [shridhar-tl/jira-assistant#367] [Q&A]
Hi
I have an error similar to https://github.com/shridhar-tl/jira-assistant/discussions/359
I can't sync outlook from JiraAssistant installed in Jira Applications TAB. I use this functionallity from Jira Extension or Jira web but, I can't use this from this type of installation.

The error is always the same 
`"could not retrieve access token from the provider 401. Error response : 
{"error":"invalid_client","error_description":"AADSTS7000215: Invalid client secret provided. Ensure the secret being sent in the request is the client secret value, not the client secret ID, for a secret added to app 'a8efd8fc-0657-490e-a622-7b2aaa4f4f46'. Trace ID: 0373cfd9-7e92-4851-a4ea-55e9a77cc500 Correlation ID: b660f147-3dbb-4593-9ed2-c840c54ab4c8 Timestamp: 2024-09-04 08:09:36Z","error_codes":[7000215],"timestamp":"2024-09-04 08:09:36Z","trace_id":"0373cfd9-7e92-4851-a4ea-55e9a77cc500","correlation_id":"b660f147-3dbb-4593-9ed2-c840c54ab4c8","error_uri":"https://login.microsoftonline.com/error?code=7000215"}"` 

Anybody can help us?

Thanks very much.

Toni

**Comment by shridhar-tl** [comment]
I hope this issue is resolved as part of #363. I hope your issue should also have been resolved by now.

**Reply by tonisc** [reply]
Hi @shridhar-tl 
it's not the same problem. My problem continue present only with Jira Assistant installed in the Atlassian  apps tab:

![imagen](https://github.com/user-attachments/assets/9a2fb5d8-fb56-4010-8b5a-1b5114dd569a)

With web or browser extension runs fine.

**Comment by shridhar-tl** [comment]
Already understood what you are facing. But the issue was resolved for Cloud apps as well last time. So thought your issue should also be resolved. Any way, while I was still not able to reproduce this issue in our instance, will still try to guess the issue. Will check and get back on this issue.
